src/GUI/UI/Books/AutonotesBook.cpp

```diff
@@ -12,22 +14,43 @@
 
 #include "GUI/GUIButton.h"
 #include "GUI/GUIFont.h"
+#include "GUI/UI/UIGame.h"
 #include "GUI/UI/Books/AutonotesBook.h"
 
 #include "Media/Audio/AudioPlayer.h"
 
 Image *ui_book_autonotes_background = nullptr;
 
+AUTONOTE_TYPE autonoteBookDisplayType;
+
+static int startingNotesIdx;
+static int currentPage;
+static int currentPageNotes;
+static std::vector<int> activeNotesIdx;
+static std::map<int, int> notesPerPage;
```

Comment:
Usual Q – can these statics go as fields into some class instead?

If not, please add a TODO.